.rubocop.yml

```diff
@@ -853,7 +853,7 @@ Lint/RequireRelativeSelfPath:
   Enabled: true
 
 Metrics/AbcSize:
-  Enabled: false
+  Enabled: true
```

Comment:
When @fallwith and I first looked at this, we thought focusing our efforts on the `lib` code would be a more valuable first step. 

To do that, you'd add something like: 
```
Exclude:
    - 'test/**/*.rb'
```